### PR TITLE
Add variable `app` and upgrade panel types

### PR DESCRIPTION
**Pull request by webwurst**
This PR fixes https://github.com/giantswarm/giantswarm/issues/20237

- Add variable `app` and upgrade panel types for the "NGINX Ingress controller" dashboard.

### Checklist

- [x] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

**Review by Gacko — APPROVED** [review]

**Comment by QuentinBisson** [comment]
Can you share a screenshot of the new dashboard ?

File: helm/dashboards/dashboards/shared/private/nginx.json [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
         "type": "dashboard"
       },
       {
-        "datasource": "default",
+        "datasource": {
+          "type": "prometheus",
+          "uid": "PBFA97CFB590B2093"
+        },
         "enable": true,
         "expr": "sum(changes(nginx_ingress_controller_config_last_reload_successful_timestamp_seconds{cluster_id=~\"$cluster\",instance!=\"unknown\",controller_class=~\"$controller_class\",namespace=~\"$namespace\"}[30s])) by (controller_class)",
```
Comment (QuentinBisson) [inline helm/dashboards/dashboards/shared/private/nginx.json:28]: Should this expr not have the app label here ?

**Comment by webwurst** [comment]
Screenshot 2022-07-19 at 11-25-42 NGINX Ingress controller - Grafana

File: helm/dashboards/dashboards/shared/private/nginx.json (hunk 1 above)
Comment (webwurst) [reply helm/dashboards/dashboards/shared/private/nginx.json]: Probably, thanks, I've added that.

File: helm/dashboards/dashboards/shared/private/nginx.json [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
+          "matchAny": false,
+          "tags": [],
+          "type": "dashboard"
+        },
         "type": "dashboard"
       },
       {
-        "datasource": "default",
+        "datasource": {
+          "type": "prometheus",
```
[the remark this answers is not in the document]
Comment (webwurst) [reply helm/dashboards/dashboards/shared/private/nginx.json]: Yes, was wondering, changed it back.

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**